### ISSUE TITLE
Line breaks causing indented text when using the theme

### ISSUE THREAD

**Issue by selimt** [rtfd/sphinx_rtd_theme#273]
When using the vertical bars to force line breaks, the text is indented 

```
| this is line 1.
| this and the previous line should be unindented.
```

results in

```
  this is line 1.
  this and the previous line should be unindented.
```

I got around this by using the raw html <br/> for now but just wanted to find out why the rtd theme was rendering the line breaks differently.

Thanks
-Selim

**Comment by tsuda7** [comment]
I ran into the same problem and am a bit hesitating to use this theme.

**Comment by Blendify** [comment]
I think I have a fix incoming

**Comment by davidfischer** [comment]
I think @selimt and @pashango2 are referring to two separate issues/concerns.

**LEFT MARGIN**
The first one is with respect to the left margin on line-blocks. The left margin is present in the [demo docs](https://sphinx-rtd-theme.readthedocs.io/en/latest/demo/demo.html#line-blocks). At the very least it needs to be there on nested line-blocks. Just to compare themes, Alabaster has no left margin on 1st level line blocks but it has a margin on nested line blocks (see screenshot). I think this is a good way forward here.

![screen shot 2018-01-15 at 10 43 20 am](https://user-images.githubusercontent.com/185043/34957163-3ad4624c-f9e1-11e7-8950-7dcd35fe1d98.png)

**BOTTOM MARGIN**
The second issue is with respect to the bottom margin on line blocks. In this specific case, it is best to look at the issue locally rather on readthedocs.org because RTD.org adds a [workaround for this specific issue](https://github.com/rtfd/readthedocs.org/blob/a7f5425/media/css/readthedocs-doc-embed.css#L23-L26) that didn't get backported into the theme. I think this issue simply needs to get backported into the theme itself. My plan is to have a bottom margin on line blocks that are not nested. I'll create a PR for it.

![screen shot 2018-01-15 at 10 46 58 am](https://user-images.githubusercontent.com/185043/34957369-356cba10-f9e2-11e7-92f9-e6f7cb89dffc.png)
(notice the margin bottom on RTD.org while there's no margin when served locally)

**Comment by davidfischer** [comment]
I co-opted @Blendify's fix (#380) and I believe that it handles both of the cases outlined above.
